server/src/main/java/io/druid/server/initialization/BaseJettyServerInitializer.java

```diff
@@ -51,4 +59,27 @@ private static void setDefaultGzipFilterHolderParameters(final FilterHolder filt
     // We don't actually have any precomputed .gz resources, and checking for them inside jars is expensive.
     filterHolder.setInitParameter("checkGzExists", String.valueOf(false));
   }
+
+  public void addExtensionFilters(ServletContextHandler handler, Injector injector) {
```

Comment:
Makes sense. All of the methods in BaseJettyServerInitializer can be made static and current derived classes can just use BaseJettyServerInitializer.func(..) instead of extending it. And, may be I will also rename it to JettyServerInitializerUtils or something.

what say @xvrl ? I can make that update in this pull request itself.